pages/linux/vncviewer.md

```diff
@@ -0,0 +1,19 @@
+# vncviewer
+
+> Launches a VNC (Virtual Network Computing) client.
+
+- Launch a VNC client which connects to a host on a given display:
+
+`vncviewer {{host}}:{{display_number}}`
+
+- Launch in full-screen mode:
+
+`vncviewer -FullScreen {{host}}:{{display_number}}`
+
+- Launch a VNC client with specific screen geometry:
```

Comment:
Perhaps .....`with a specific`..... would read better?

Reply:
@sbrl yes I think so, I will make the change.